Formula/baidupcs-go.rb

```diff
@@ -0,0 +1,16 @@
+class BaidupcsGo < Formula
+  desc "百度网盘客户端 - Go语言编写"
+  homepage "https://github.com/iikira/BaiduPCS-Go"
+  url "https://github.com/iikira/BaiduPCS-Go/archive/v3.6.1.tar.gz"
+  sha256 "07e1d802e6b49425a53b883c354d8b753de55f7005ac8c7b27cd8396db0447da"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-o", bin/"baidupcs-go"
+  end
+
+  test do
+    system bin/"baidupcs-go", "--version"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Fixed